src/messages/MOSDPGRecoveryDelete.h

```diff
@@ -53,7 +53,8 @@ struct MOSDPGRecoveryDelete : public MOSDFastDispatchOp {
       from(from),
       pgid(pgid),
       map_epoch(map_epoch),
-      min_epoch(min_epoch) {}
+      min_epoch(min_epoch),
+      cost(0) {}
 
```

Comment:
I would prefer `cost` to be initialized at the declaration itself. Otherwise, another overloading constructor may create the same problem (coverity unhappy) again in the future :-).